readthedocs/proxito/views/hosting.py

```diff
@@ -489,12 +489,13 @@ def _v1(self, project, version, build, filename, url, request):
                         # to the user's query.
                         # It uses "Search query sintax":
                         # https://docs.readthedocs.io/en/stable/server-side-search/syntax.html
-                        # [
-                        #     "Include subprojects",
-                        #     f"subprojects:{project.slug}/{version.slug}",
-                        # ],
+                        [
+                            "Include subprojects",
+                            f"subprojects:{project.slug}/{version.slug}",
+                        ],
                     ]
-                    if version
+                    # Show the subprojects filter on the parent project and subproject
+                    if project.subprojects.exists() or project.superprojects.exists()
```

Comment:
For this filter to work on subprojects, the parent project must be used in the filter `subprojects:{parent-project-slug}`

Reply:
@stsewd Updated, if you could give it another peek ❇️